lib/swagger-definition.js

```diff
@@ -20,16 +20,29 @@ const path = require('path')
 const packagePath = path.join(process.cwd(), 'package.json')
 const { name, description, version } = require(packagePath)
 
-module.exports = function swaggerDefinition(customDefinition) {
+module.exports = function swaggerDefinition(customDefinition, openApiSpecification) {
+  // If customDefinition is set and contains other than only openApiSpecification, it is returned without that field.
+  if (customDefinition) {
+    delete customDefinition.openApiSpecification
+    if (Object.keys(customDefinition).length > 0) {
+      return customDefinition
+    }
+  }
+
   const defaultDefinition = {
     info: {
       title: name,
       description,
       version,
     },
-    consumes: ['application/json'],
-    produces: ['application/json'],
   }
-
-  return customDefinition || defaultDefinition
+  switch (openApiSpecification) {
+  case 'swagger' :
+    defaultDefinition.consumes = ['application/json']
+    defaultDefinition.produces = ['application/json']
+    break
+  default:
+    break
+  }
+  return defaultDefinition
```

Comment:
before this change `customDefinition` was returned, right now only defaultDefinition will returned, why this change?

Reply:
Before, `customDefinition` was a complete definition or undefined. Now it can even just contain the `openApiSpecification`. 
For this reason I changed the flow of this function, and `customDefinition` is returned at [line 28](https://github.com/mia-platform/lc39/pull/119/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-8102913d96b950622f48cd3e79b629f33fc53b80af593df7e4ac34af4dcd8645R28), only if it contains other than just `openApiSpecification`

Reply:
@fredmaggiowski Can we close this discussion?

Reply:
@fredmaggiowski up

Reply:
could this be a breaking change?

Reply:
@davidebianchi What do you think about this discussion? Can we close this?

Reply:
Yes, it is a breaking change for the exposed api. This will bring to v5 of lc39.